### ISSUE TITLE
So which versions do we support?

### ISSUE THREAD

**Issue by sole** [mozilla/node-firefox-find-simulators#19]
judging by the comments in https://github.com/mozilla/node-firefox-find-simulators/issues/5 we might support 1.2+ ? if that's the case, shouldn't we document it on the README under "current limitations"?

**Comment by tofumatt** [comment]
Yeah, I think 1.2+ is the general target too, my understanding was debugging/interacting with 1.0/1.1 devices is a lot messier/worse.

I'll file a PR for the README, but I think in general any module in node-firefox working on anything pre-1.2 is a nice bonus but not a requirement.
